lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -1609,14 +1653,15 @@ describe('AwsCompileFunctions', () => {
 
       const expectedOutputs = {};
 
-      awsCompileFunctions.compileFunctions();
-
-      expect(
-        awsCompileFunctions.serverless.service.provider.compiledCloudFormationTemplate
-          .Outputs
-      ).to.deep.equal(
-        expectedOutputs
-      );
+      return expect(awsCompileFunctions.compileFunctions()).to.be.fulfilled
+      .then(() => {
+        expect(
+          awsCompileFunctions.serverless.service.provider.compiledCloudFormationTemplate
+            .Outputs
+        ).to.deep.equal(
+          expectedOutputs
+        );
+      });
     });
   });
```

Comment:
To myself: Add unit test that checks for version resource after configuration change